src/bundle/Resources/public/js/scripts/admin.context.menu.js

```diff
@@ -0,0 +1,53 @@
+(function(global, doc, eZ) {
+    const adapatItemsContainer = doc.querySelector('.ibexa-context-menu');
+
+    if (adapatItemsContainer) {
+        const menuButtons = [...adapatItemsContainer.querySelectorAll('.ibexa-context-menu__item > .ibexa-btn:not(.ibexa-btn--more)')];
+        const popupMenuElement = adapatItemsContainer.querySelector('.ibexa-popup-menu');
+        const showPopupButton = adapatItemsContainer.querySelector('.ibexa-btn--more');
+        const adaptiveItems = new eZ.core.AdaptiveItems({
+            items: [...adapatItemsContainer.querySelectorAll('.ibexa-context-menu__item:not(.ibexa-context-menu__item--more)')],
```

Comment:
You can add a special class to the items and AdaptiveItems will find items on its own.